plugins/pencilblue/controllers/actions/admin/sites/delete_site.js

```diff
@@ -30,12 +30,13 @@ module.exports = function DeleteSiteActionModule(pb) {
         var self = this;
         var siteid = self.pathVars.siteid;
         var siteQueryService = new pb.SiteQueryService();
+        var SiteService = new pb.SiteService();
         siteQueryService.getCollections(function(err, allCollections) {
-            siteQueryService.deleteSiteSpecificContent(allCollections, siteid, function(err, result) {
-                if(util.isError(err)) {
+            SiteService.deleteSiteSpecificContent(allCollections, siteid, function(err, result) {
```

Comment:
Why not call the code shown below instead?

```
siteService.deleteSingle({ where: { uid: siteid}}, function(err, site) {
  ...
});
```
The above implies that the SiteService registers a before or after delete event to remove site specific content.  

* Is it really important to get the number of records deleted?